stack/stack_mac_gnu.yaml

```diff
@@ -0,0 +1,359 @@
+gnu:
```

Comment:
Can you say how this version differs specifically for GNU, compared to stack_mac?

Reply:
I did not use stack_mac.yaml, maybe it could be reserved for using it with clang+gfortran. Yet to be tested with this combination.